### ISSUE TITLE
Will there be an update for Pajarito.jl to work with JuMP 0.19?

### ISSUE THREAD

**Issue by erikfilias** [JuliaOpt/Pajarito.jl#424]
Hi, I am using Pajarito.jl to solve misdp problems. Everything is ok until JuMP v0.18.5, however there an new update of JuMP to version 0.19. I tried to use Pajarito with JuMP V0.19 but have the following error:
```
ERROR: Unsatisfiable requirements detected for package Pajarito [2f354839]:
 Pajarito [2f354839] log:
 ├─possible versions are: [0.1.0-0.1.1, 0.2.0-0.2.1, 0.3.0-0.3.2, 0.4.0-0.4.2, 0.5.0-0.5.1, 0.6.0-0.6.1] or uninstalled
 ├─restricted to versions 0.6.1 by an explicit requirement, leaving only versions 0.6.1
 └─restricted by compatibility requirements with JuMP [4076af6c] to versions: [0.1.0-0.1.1, 0.2.0-0.2.1, 0.3.0-0.3.2, 0.4.0-0.4.2, 0.5.0-0.5.1, 0.6.0] or uninstalled — no versions left
   └─JuMP [4076af6c] log:
     ├─possible versions are: 0.19.0 or uninstalled
     └─JuMP [4076af6c] is fixed to version 0.19.0+
```
Will there be an update for Pajarito.jl to work with JuMP 0.19?

**Comment by chriscoey** [comment]
Hi, yes there will be an update for MOI and JuMP 0.19 in the next few months. We are just waiting for some progress on MIP callbacks and MathOptFormat files first.

**Comment by erikfilias** [comment]
> Hi, yes there will be an update for MOI and JuMP 0.19 in the next few months. We are just waiting for some progress on MIP callbacks and MathOptFormat files first.

Thank you so much,
Best regards,

**Comment by chriscoey** [comment]
An update on this: still waiting on MIP callbacks in the new JuMP 0.19.

**Comment by jay-dave** [comment]
Hi, Is there any news on this update? Thank you!

**Comment by chriscoey** [comment]
@jay-dave yes there has been progress in MOI - https://github.com/JuliaOpt/MathOptInterface.jl/pull/782
once that is merged and callbacks are available for JuMP then Pajarito can be updated for MOI.
I would not expect it to be ready in the next month though, sorry.

**Comment by jay-dave** [comment]
@chriscoey Thanks for the information.

**Comment by ericphanson** [comment]
https://github.com/JuliaOpt/MathOptInterface.jl/pull/782 was merged yesterday! I am excited for MOI Pajarito (so we don't have to write a MPB wrapper for [SDPAFamily.jl](https://github.com/ericphanson/SDPAFamily.jl) to try it out with Pajarito).

**Comment by odow** [comment]
Closing in favor of #411
